internal/cli/test.go

```diff
@@ -360,3 +356,48 @@ func (c *cli) customDomainPickerOptions() (pickerOptions, error) {
 
 	return opts, nil
 }
+
+func (c *cli) appPickerWithCreateOption() (pickerOptions, error) {
+	options, err := c.appPickerOptions()
+	if err != nil {
+		return nil, err
+	}
+
+	enhancedOptions := []pickerOption{
+		{
+			value: NEW_CLIENT,
+			label: "Create a new client to use for testing the login",
+		},
+	}
+	enhancedOptions = append(enhancedOptions, options...)
+
+	return enhancedOptions, nil
+}
+
+func checkClientIsAuthorizedForAPI(cli *cli, client *management.Client, audience string) error {
+	var list *management.ClientGrantList
+	if err := ansi.Waiting(func() (err error) {
+		list, err = cli.api.ClientGrant.List(
+			management.Parameter("audience", audience),
+			management.Parameter("client_id", client.GetClientID()),
+		)
+		return err
+	}); err != nil {
+		return fmt.Errorf(
+			"failed to find client grants for API identifier %q and client ID %q: %w",
+			audience,
+			client.GetClientID(),
+			err,
+		)
+	}
+
+	if len(list.ClientGrants) < 1 {
+		return fmt.Errorf(
+			"the %s application is not authorized to request access tokens for this API %s",
```

Comment:
```suggestion
			"the %s application is not authorized to request access tokens for the %s API. To resolve, either issue a client grant through the Management API or authorize the API for the application in the dashboard.",
```

Reply:
Fixed in [`2890b03` (#629)](https://github.com/auth0/auth0-cli/pull/629/commits/2890b03c0f7481acd1e53d1612e044c760a16138), but I suggested to run auth0 apps open <client-id> instead.